bin/varnishtest/tests/c00018.vtc

```diff
@@ -123,11 +123,13 @@ logexpect l1 -v v1 -g raw {
 	expect 0 1011	VCL_call        {^HASH$}
 	expect 0 1011	VCL_return      {^lookup$}
 	expect 0 1011	Timestamp       {^Process:}
-	expect 0 1011	RespHeader      {^Date:}
-	expect 0 1011	RespHeader      {^Server: Varnish$}
-	expect 0 1011	RespHeader      {^X-Varnish: 1011$}
 	expect 0 1011	RespProtocol    {^HTTP/1.1$}
 	expect 0 1011	RespStatus      {^405$}
+	expect 0 1011	RespReason      {^Method Not Allowed$}
+	# XXX dup RespReason
```

Comment:
Why is this a duplicate? If you set the status it will update the reason if it's a status code Varnish knows. I think you simply ran into this log record because the order was changed where before the test stopped looking after `RespStatus`.

I think this comment is safe to remove.

Reply:
Yes, this existed before and after, I just meant to mark the fact that we might want to have a look later.